modules/apps/content-dashboard/content-dashboard-blogs-impl/src/main/java/com/liferay/content/dashboard/blogs/internal/item/action/provider/ViewBlogsEntryContentDashboardItemActionProvider.java

```diff
@@ -60,7 +60,7 @@ public String getKey() {
 	public boolean isShow(
 		BlogsEntry blogsEntry, HttpServletRequest httpServletRequest) {
 
-		if (!blogsEntry.isVisible()) {
+		if (blogsEntry.isDraft() || blogsEntry.isInTrash()) {
```

Comment:
Hey @YurenaC !!!

I needed to change this code, because this is the condition we use in [com.liferay.blogs.web.internal.layout.display.page.BlogsLayoutDisplayPageProvider.](https://github.com/liferay/liferay-portal/blob/c1e0906fb1708bacf5bba152dc986b618c50d489/modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/layout/display/page/BlogsLayoutDisplayPageProvider.java#L49)

With the _**isVisible**_ condition, the display page templates could not be rendered, so a null pointer exception was obtained when the Content Performance Panel was painted.

Maybe the Lima Team could that into account for the isVIsible method, I could ask them in the lima channel.

Thanks!!

Reply:
Great, thanks 👍